src/components/anti-everyone.ts

```diff
@@ -31,9 +45,58 @@ export default class AntiEveryone extends BotComponent {
         if (message.content.match(failed_everyone_re) != null) {
             // NOTE: .toLocaleString("en-US") formats this number with commas.
             const memberCount = this.wheatley.TCCPP.members.cache.size.toLocaleString("en-US");
-            await message.reply({
+
+            // Store the reply for later deletion, if necessary
+            if (!this.replies.has(message.author)) {
+                this.replies.set(message.author, []);
+            }
+            const reply = await message.reply({
                 content: `Did you really just try to ping ${memberCount} people?`,
             });
+
+            // Store the reply for later deletion, along with the message it was replying to
+            this.replies.get(message.author)!.push(
+                Object.assign(
+                    {
+                        replyTo: message.id,
+                    },
+                    reply,
+                ),
+            );
```

Comment:
I think it'd be better to just store `{message: Discord.Message; replied_to: string}` structs rather than this. I'm somewhat weary of Object.assigning a message to another object like this since I don't know if it could lure out subtle unexpected behaviors from discordjs.

Reply:
I've done this in the past without issues, but I do see your point, so I've updated the code to match the `{message: Discord.Message; replied_to: Discord.Message["id"]}` objects

Reply:
Thanks, I appreciate it. It's good to know you have found it to be ok in the past.